.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: |
           cargo tree
           git checkout Cargo.lock
-          cargo tree --frozen
+          cargo tree
```

Comment:
https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/240/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R144

When we update yellowstone-grpc to later package we start getting dependency conflicts. We will be looking at resolving this but would like to do in a follow up.